scripts/chronic_disease_indicators_in_us__cdi_.json

```diff
@@ -0,0 +1,186 @@
+{
+    
+    "citation": "Centers for Disease Control and Prevention",
+    "description": "DC's Division of Population Health provides cross-cutting set of 124 indicators that were developed by consensus and that allows states and territories and large metropolitan areas to uniformly define, collect, and report chronic disease data that are important to public health practice and available for states, territories and large metropolitan areas. In addition to providing access to state-specific indicator data, the CDI web site serves as a gateway to additional information and data resources.CDC's Division of Population Health provides cross-cutting set of 124 indicators that were developed by consensus and that allows states and territories and large metropolitan areas to uniformly define, collect, and report chronic disease data that are important to public health practice and available for states, territories and large metropolitan areas. In addition to providing access to state-specific indicator data, the CDI web site serves as a gateway to additional information and data resources.",
```

Comment:
"description": "124 indicators by CDC's Division of Population Health used to uniformly define, collect, and report chronic disease data"